packages/app/src/specs/SpecsList.vue

```diff
@@ -111,13 +209,32 @@
       @clear="handleClear"
     />
   </div>
+  <LoginModal
+    v-model="isLoginOpen"
+    :gql="props.gql"
+    :utm-medium="loginUtmMedium"
+    @loggedin="refreshPage"
+  />
+  <CloudConnectModals
+    v-if="isProjectConnectOpen"
+    :gql="props.gql"
+    @cancel="isProjectConnectOpen = false"
+    @success="isProjectConnectOpen = false; refreshPage()"
```

Comment:
Just to keep the implementation of these two modals in alignment, is the `isProjectConnectOpen` update necessary here, or will the `refreshPage` call take care of it the same way we're doing on `LoginModal` above?